nebula/src/nebula/video_storage/routes/video_routes.py

```
@@ -0,0 +1,248 @@
+
+        return UploadResponse(
+            video_id=metadata.video_id,
+            filename=metadata.filename,
+            size_bytes=metadata.size_bytes,
+            uploaded_at=metadata.uploaded_at,
+            playlist_url=playlist_url,
+            duration_seconds=metadata.duration_seconds,
+        )
+
+    except Exception as e:
+        logger.error("Error saving video: %s", e)
```

Ruff (0.14.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)